apps/api/scripts/award_boost.ts

```diff
@@ -2,7 +2,8 @@ import * as dotenv from "dotenv";
 import cron from "node-cron";
 import * as path from "path";
 
-import { BlockchainAddressAsU8A } from "@/lib/coders.js";
+import { BlockchainAddressAsU8A } from "@recallnet/db-schema/coders";
```

Comment:
Moved the coders to the shared db packages so they can be used in that package and here in the api app.